docs/context-and-outputs.md

```diff
@@ -0,0 +1,129 @@
+
+**URLScan**
+
+```python
+
+    dbot_score = {
+        'Indicator': 'malware.wicar.org',
+        'Score': 3,
+        'Type': 'url'
+    }
+    
+    url_cont = {
+        'Data': 'malware.wicar.org',
+        'Malicious': {
+            'Description': 'Match found in Urlscan.io database',
+            'Vendor': 'urlscan.io'
+        }
+    }
+    
+    cont = {
+        'ASN': 'AS40630',
+        'Certificates': {
+            'Issuer': 'Google Internet Authority G3',
+            'SubjectName': '*.googleapis.com',
+            'ValidFrom': '2018-12-04 09:33:00',
+            'ValidTo': '2019-02-26 09:33:00'
+        },
+        'Country': ['US', 'EU', 'IE', 'DE', 'NL'],
+        'Data': 'malware.wicar.org',
+    }
+
+    ec = {
+        'URLScan(val.URL && val.URL == obj.URL)': cont, # Please see the section on DT below
+        'DBotScore': dbot_score,
+        'URL': url_cont
+    }
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['markdown'],
+        'Contents': response,
+        'HumanReadable': tableToMarkdown('{} - Scan Results'.format(url_query), human_readable),
+        'EntryContext': ec
+    })
+```
+
+In this example, we are pushing three different dictionaries into the context. ```cont``` which contains the context specific to URLScan, ```dbot_score``` which contains the DBot score context, and ```URL``` which is part of the context standard for URLs.
+
+In the Integration settings, we see:
+
+![screen shot 2019-01-09 at 14 54 40](doc_imgs/50900672-90a7c800-141e-11e9-946d-8d5352e279ec.png)
+
+These paths can be seen in the context found in the war room, here:
+
+![screen shot 2019-01-09 at 14 57 17](doc_imgs/50900776-e3817f80-141e-11e9-8263-d75cf6cf2dc0.png)
+
+
+**Please Note:** When improving or replacing an existing integration, verify that the context outputs are backwards compatible. Otherwise, changes can break playbooks and prevent them from running.
+
+## Context
+Context is defined by an integration by adding the following to the results command:
+```
+ec = 'VirusTotal.IP(val.Address==obj.Address)': {
+    'Address': '8.8.8.8'.,
+    'ASN': 'BLA_ASN',
+    'Domain': ['google.com']
+}
+
+demisto.results({
+    'Type': entryTypes['note'],
+    'ContentsFormat': formats['markdown'],
+    'Contents': raw_content_obj,
+    'HumanReadable': human_readable_obj,
+    'EntryContext': ec
+})
+```
+
+## DT (Demisto Transform Language)
+In the above example, we observe the entry context using ```(val.Address==obj.Address)```. This works to *tie together* related entry context objects. In this instance, we are using the value of the Address key as the unique identifier to search through the existing context and link related objects. This prevents data from being overwritten as well as further enriches an existing entry with more information. Learn more about linking context [here](https://github.com/demisto/content/blob/master/docs/code_conventions/README.MD#linking-context).
+
```

Comment:
Link is pointing to the demisto/content GitHub repo

Reply:
fixed